docs/swupd.1.rst

```diff
@@ -51,165 +51,121 @@ OPTIONS
 The following options are applicable to most subcommands, and can be
 used to modify the core behavior and resources that swupd uses.
 
--  ``-h, --help``
+-h, --help    Display general help information. If put after a subcommand, it
+        will display help specific to that subcommand.
 
-   Display general help information. If put after a subcommand, it will
-   display help specific to that subcommand.
+-v, --version   Displays the version information of the swupd program, and exit.
```

Comment:
I have one comment that is repeated many times in this PR, although I am not 100% sure it is part of this PR or if it should be done in a separate PR. But we have a lot of inconsistencies where some flags that expect a value sometimes they show the value and some time they don't. I think all those should always show a value, for example "version" in this case. I think it should be:
`-v <version number>, --version=<version number> `

the same applies for example to  `-C <path>, --certpath <path>`, `-W <num of downloads>, --max-parallel-downloads <num of downloads>`, etc.

Reply:
It's part of this patch. I'll review that. thanks for pointing this out

Reply:
The --version is correc the way it is because there's no parameters expected.
I fixed all other cases I found. If there's anything wrong left, please comment here.

Reply:
haha, true, I chose a bad example because this --version does not expect parameters as you said 😝